python/tests/test_extracted_datatypes.py

```diff
@@ -198,25 +199,33 @@ def test_to_dict(self,
 class TestCeiling:
 
     @pytest.fixture
-    def sample(self) -> typing.Dict[str, typing.Any]:
-        return {
-            'label': 'Main attic',
-            'typeEnglish': 'Attic/gable',
-            'typeFrench': 'Combles/pignon',
-            'nominalRsi': 2.864,
-            'effectiveRsi': 2.9463,
-            'area': 46.4515,
-            'length': 23.875,
-        }
-
-    def test_from_data(self, sample: typing.Dict[str, typing.Any]):
+    def sample(self) -> etree._Element:
+        data = """
+<Ceiling>
+    <Label>Main attic</Label>
+    <Construction>
+        <Type>
+            <English>Attic/gable</English>
+            <French>Combles/pignon</French>
+        </Type>
+        <CeilingType nominalInsulation='2.864' rValue='2.9463' />
+    </Construction>
+    <Measurements area='46.4515' length='23.875' />
+</Ceiling>
+        """
+        return etree.fromstring(data)
+
+    def test_from_data(self, sample: str):
```

Comment:
Isn't `sample` an `etree._Element`, not `str`?

Reply:
yes, good catch. That's me trying to catch the typing issues.